packages/components/pages/f-account-info/test-utils/component-objects/f-account-info-selectors.js

```diff
@@ -0,0 +1,46 @@
+exports.COMPONENT = '[data-test-id="account-info"]';
+exports.CHANGE_EMAIL_ADDRESS_LINK = '[name="account-info-change-emailAddress-link"]';
```

Comment:
My only worry with relying on the `name` property is that if it changes in the future, it's not immediately clear that it will cause the tests to break.

IMO we should be consistent in using `data-test-id`. If we're using `name` elsewhere in the repo, we should look to change these back in future PR's 👍🏼

Reply:
yep I agree actually, probably worth making a ticket to revert `f-form-field` so it no longer has this function and we can simply add the `data-test-id` in